dspace-api/src/test/data/dspaceFolder/config/local.cfg

```diff
@@ -192,4 +192,14 @@ ldn.notify.inbox.block-untrusted-ip = true
 # ERROR LOGGING                           #
 ###########################################
 # Log full stacktrace of other common 4xx errors (for easier debugging of these errors in tests)
-logging.server.include-stacktrace-for-httpcode = 422, 400
\ No newline at end of file
+logging.server.include-stacktrace-for-httpcode = 422, 400
+
+### JCloudSettings
+
+# Enabled the JCloudStore
+assetstore.jcloud.enabled = true
+assetstore.jcloud.useRelativePath = false
+assetstore.jcloud.subfolder = assetstore
+assetstore.jcloud.provider = filesystem
+assetstore.jcloud.container = assetstore-jclouds-container
```

Comment:
I'm a little nervous about setting these test configs *globally*, as it can sometimes result in side effects for other tests (especially in scenarios where tests are run in a different order).  Are these even necessary? Maybe I'm overlooking it, but it seems like these may not be used in your tests.

If you do need them in tests, the ideal scenario is to set them per test by doing `configurationService.setProperty(name, value);`  That will set the property for the remainder of your tests.

For example, see how this is achieved for `S3BitStoreServiceIT`, which is similarly setting these values for S3 tests.

Reply:
As these are only used to initialize the JCloud assetstore to use the filesystem provider and place the files in the local target directory while testing, there should not be impacts due to ordering of other tests.

This configuration is used for the BitstreamJCloudBitstoreTest that ensures that the JCloudsBitstore provider generates expected results when enabled and actual content is added/removed from the storage api.